package/yast2-network.changes

```diff
@@ -1,4 +1,41 @@
 -------------------------------------------------------------------
+Mon Apr 27 07:06:18 UTC 2015 - mfilka@suse.com
+
+- bnc#912904 
+  - do not crash when adding new device and proposed configuration
+    name is changed to new one
+- bnc#914833
+  - do not crash when renaming virtual device via hardware tab
+- bnc#910337
+  - initialize configuration name in Hardware tab correctly. It
+    means according device type. Also uses custom name if user
+    used nonstandard one.
+- bnc#905738
+  - udev rules provided by AutoYaST profile gets applied
+- bnc#918356
+  - populate bond slave candidates list with proper device names
+- 3.1.114
```

Comment:
@kobliha Yes, you're right, we had this version in the past but the code is exactly the same (the only difference is the Rakefile). Anyway, as latest yast2-network version in this branch is 3.1.110, we could tag this version as 3.1.110.1 or something like that.